### ISSUE TITLE
Treat tilde as whitespace.

### ISSUE THREAD

**Issue by reitermarkus** [antonWetzel/typst-languagetool#1]
I often use tilde so that citations are not wrapped to the next line, e.g.

```
info[MORFOLOGIK_RULE_EN_GB]: Possible spelling mistake
  --> main.typ:63:21
   |
63 | PP") as well as the Rust~@rust programming la
   |                     ----- info: Possible spelling mistake found.
   |                             help: Rust
   |                             help: Rusty
   |                             help: Rusts
   |                             help: RUSTA
   |                             help: Rust X
```

**Comment by antonWetzel** [comment]
Fixed with 87b6f4445405b9898b46e80c2fb82d814a6135c3. The tilde shorthand is replaced with a simple space.

**Comment by reitermarkus** [comment]
Thanks for the quick fix!
